### DISCUSSION TITLE
Dropdown button layout

### DISCUSSION THREAD

**Discussion by CrawfordW** [quasarframework/quasar#11943] [General - Components / Directives / etc]
Is there any way to get q-btn-dropdown to look:
![Dropdown Button](https://user-images.githubusercontent.com/45148066/148575389-5d1241bb-a7f3-4aed-b81a-280be7b06fca.png)
I can't figure out how to stack the label under the main icon without also stacking the caret.

So I'm having to make my own custom component which seems a lot of unnecessary work.

**Comment by pdanpdan** [comment]
Please make a codepen with what you have tried

**Reply by CrawfordW** [reply]
Never mind. I decided to keep my custom component (just combines a q-btn and q-menu). This also gives me some default behaviors and some extra props for consistency throughout my app.
Thanks. Quasar is awesome!
